pkg/querier/iterators/chunk_merge_iterator.go

```diff
@@ -78,59 +88,87 @@ outer:
 	return its
 }
 
-func (c *chunkMergeIterator) Seek(t int64) bool {
+func (c *chunkMergeIterator) Seek(t int64) chunkenc.ValueType {
 	c.h = c.h[:0]
 
+	if c.currErr != nil {
```

Comment:
I think should be moved above the line `c.h = c.h[:0]`

Reply:
fixing